.github/workflows/dotnettest.yml

```diff
@@ -0,0 +1,20 @@
+name: .NET Core
+
+on:
+  push:
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-16.04, ubuntu-18.04]
```

Comment:
Just test only on Ubuntu 18.04? I feel it does not make sense to test on Ubuntu 16.04 anymore.